docs/api.rst

```diff
@@ -0,0 +1,267 @@
+.. _api:
+
+API
+###
+
+The BuildHub API is just a Kinto instance with a collection of records. A :ref:`series of jobs <jobs>` is charge of keeping those records up to date when new releases are published.
+
+Servers
+=======
+
+* **Development**: https://kinto-ota.dev.mozaws.net/v1
+* **Stage**: https://kinto-ota.dev.mozaws.net/v1
+* **Production**: https://kinto-ota.dev.mozaws.net/v1
+
+Clients
+=======
+
+Since it is an HTTP API and the records are public, you can simply use any HTTP client, like `curl <http://curl.haxx.se>`_ or `httpie <https://httpie.org>`_.
+
+But for more convenience, especially regarding pagination and error handling, some dedicated libraries are also available:
+
+* `kinto-http.js <https://github.com/Kinto/kinto-http.js>`_ (JavaScript)
+* `kinto-http.py <https://github.com/Kinto/kinto-http.py>`_ (Python)
+* `kinto-http.rs <https://github.com/Kinto/kinto-http.rs>`_ (Rust)
+* `kinto-http.java <https://github.com/intesens/kinto-http-java>`_ (Java)
+
+Data
+====
+
+A single record has the following fields (:ref:`see below <data-schema>` for more details):
+
+.. code-block:: javascript
+
+    {
+        "id": "firefox_beta_50-0b11_macosx_el",
+        "schema": 1497453926485,
+        "last_modified": 1498140377629,
+        "target": {
+            "platform": "macosx",
+            "os": "mac",
+            "version": "50.0b11",
+            "channel": "beta",
+            "locale": "el"
+        },
+        "build": {
+            "id": "20161027110534",
+            "date": "2016-10-27T11:05:34Z",
+            "number": 2,
+            "as": "$(CC)",
+            "ld": "ldd",
+            "cc": "/usr/bin/ccache /home/worker/workspace/build/src/gcc/bin/gcc -std=gnu99",
+            "cxx": "/usr/bin/ccache /home/worker/workspace/build/src/gcc/bin/g++ -std=gnu++11",
+            "host": "x86_64-pc-linux-gnu",
+            "target": "x86_64-pc-linux-gnu",
+        },
+        "source": {
+            "tree": "releases/mozilla-beta",
+            "product": "firefox",
+            "revision": "829a3f99f2606759305e3db204185242566a4ca6",
+            "repository": "https://hg.mozilla.org/releases/mozilla-beta"
+        },
+        "download": {
+            "mimetype": "application/x-apple-diskimage",
+            "date": "2016-10-28T00:56:42Z",
+            "size": 86180614,
+            "url": "https://archive.mozilla.org/pub/firefox/releases/50.0b11/mac/el/Firefox 50.0b11.dmg"
+        }
+    }
+
+
+Listing records
+===============
+
+Basic API
+---------
+
+The list of records is available at:
+
+* `$SERVER/buckets/build-hub/collections/releases/records <https://kinto-ota.dev.mozaws.net/v1/buckets/build-hub/collections/releases/records?_limit=10>`_
+
+And a single record at:
+
+* `$SERVER/buckets/build-hub/collections/releases/records/${ID} <https://kinto-ota.dev.mozaws.net/v1/buckets/build-hub/collections/releases/records/firefox_beta_50-0b11_macosx_el>`_
```

Comment:
`${SERVER}` to match `${ID}`? Or vice versa?

Reply:
Didn't get that...

Reply:
Both `$SERVER` and `${ID}` are variable interpolations, but they have different syntax and I don't think they need to, so I'd suggest changing them both to `$FOO` or both to `${FOO}`.

Reply:
Ohhh I wasn't awake enough! Thank you!